Sources/EdgeNetworkHandlers/NetworkResponseHandler.swift

```diff
@@ -152,7 +154,13 @@ class NetworkResponseHandler {
             if ignoreStorePayloads {
                 Log.debug(label: LOG_TAG, "Identities were reset recently, ignoring state:store payload for request with id: \(requestId)")
             } else {
-                handleStoreEventHandle(handle: eventHandle)
+                if let type = eventHandle.type {
+                    if EdgeConstants.JsonKeys.Response.EVENT_HANDLE_TYPE_STORE == type.lowercased() {
```

Comment:
nit: Is `type.lowercased()` required? Calling it only for `if` condition is little confusing.

Reply:
I don't think it's required as the response uses "state:store" (lowercased). However, the original code used it so I preserved the logic.  The handle type "locationHint:result" is mix cased so I'm doing an exact match.